### PR TITLE
Adds a Chameleon belt

### PR DESCRIPTION
**Pull request by Supermichael777**
:cl:
add: Wannabe ninjas have been found carrying an experimental chameleon belt. The Spider clan has disavowed any involvement.
/:cl:

The assault belt was a one way trip to getting busted in a stealth kit.  This is slightly better. Go ninja go ninja go!

### CODE REVIEW

File: code/modules/clothing/chameleon.dm [hunk 1]
```diff
@@ -437,6 +437,22 @@
 /obj/item/weapon/storage/backpack/chameleon/emp_act(severity)
 	chameleon_action.emp_randomise()
 
+/obj/item/weapon/storage/belt/chameleon
+	name = "toolbelt"
+	desc = "Holds tools."
```
Comment (coiax) [inline code/modules/clothing/chameleon.dm]: make sure that it looks like a real item in its default state, otherwise it's a metatell

**Comment by coiax** [comment]
Also, isn't the combat belt purchasable by regular traitors?

**Comment by AdamElTablawy** [comment]
Yeah. But it's basically a 'VALID ME' sign for some extra storage.

**Comment by Supermichael777** [comment]
@coiax it inherits tool-belt sprites

**Comment by coiax** [comment]
@Supermichael777 and name and description

**Comment by Supermichael777** [comment]
@coiax did you look at the code? I set them to default to the toolbelt. Their is never a point where you could tell it apart. This is merge ready

**Comment by jughu** [comment]
>ninja kit a fucking strong katana with high blockchance that also fit in the backpack
>a chameleon belt stealth storage for their ninja stars and cham projector
>stop buffing fucking ninja bundle

**Pull request merged**